README.md

```diff
@@ -89,13 +89,16 @@ To contribute to the development of xDSL follow the subsequent steps.
 
 ### Developer Installation
```

Comment:
Can we keep a subsection that just gives the commands for the standard pip installation?
Even though we want to make `uv` the "default", I feel it's important to keep compatibility with `pip`.
We just need to write the `python -m venv venv`, and `source venv/bin/activate`, the `pip install -e .`, and explain that we don't need the `uv run` prefixes. in the rest of the documentation.
That way, we always have a fallback, especially for people already using pip/venv.